docs/src/docs/asciidoc/customizing-requests-and-responses.adoc

```diff
@@ -3,68 +3,74 @@
 
 There may be situations where you do not want to document a request exactly as it was sent
 or a response exactly as it was received. Spring REST Docs provides a number of
-preprocessors that can be used to modify a request or response before it's documented.
+preprocessors that can be used to modify a request or response before it is documented.
 
-Preprocessing is configured by calling `document` with an `OperationRequestPreprocessor`,
-and/or an `OperationResponsePreprocessor`. Instances can be obtained using the
-static `preprocessRequest` and `preprocessResponse` methods on `Preprocessors`. For
-example:
+Preprocessing is configured by calling `document` with an `OperationRequestPreprocessor`
+or an `OperationResponsePreprocessor`. You  an obtain instances by using the
+static `preprocessRequest` and `preprocessResponse` methods on `Preprocessors`. The
+following examples show how to do so:
 
+====
 [source,java,indent=0,role="primary"]
 .MockMvc
 ----
 include::{examples-dir}/com/example/mockmvc/PerTestPreprocessing.java[tags=preprocessing]
 ----
-<1> Apply a request preprocessor that will remove the header named `Foo`.
-<2> Apply a response preprocessor that will pretty print its content.
+<1> Apply a request preprocessor that removes the header named `Foo`.
+<2> Apply a response preprocessor that pretty prints its content.
 
 [source,java,indent=0,role="secondary"]
 .WebTestClient
 ----
 include::{examples-dir}/com/example/webtestclient/PerTestPreprocessing.java[tags=preprocessing]
 ----
-<1> Apply a request preprocessor that will remove the header named `Foo`.
-<2> Apply a response preprocessor that will pretty print its content.
+<1> Apply a request preprocessor that removes the header named `Foo`.
+<2> Apply a response preprocessor that pretty prints its content.
 
 [source,java,indent=0,role="secondary"]
 .REST Assured
 ----
 include::{examples-dir}/com/example/restassured/PerTestPreprocessing.java[tags=preprocessing]
 ----
-<1> Apply a request preprocessor that will remove the header named `Foo`.
-<2> Apply a response preprocessor that will pretty print its content.
+<1> Apply a request preprocessor that removes the header named `Foo`.
+<2> Apply a response preprocessor that pretty prints its content.
+====
 
 Alternatively, you may want to apply the same preprocessors to every test. You can do
-so by configuring the preprocessors using the `RestDocumentationConfigurer` API in your
+so by configuring the preprocessors to use the `RestDocumentationConfigurer` API in your
```

Comment:
I have problems with stringing together gerunds, because it becomes impossible to follow.

Can it be as follows? "You can do so by using the `RestDocumentationConfigurer` API in your `@Before` method to configure the preprocessors."

Reply:
It can indeed.